### ISSUE TITLE
Installation issue: flux-sched

### ISSUE THREAD

**Issue by PhilipFackler** [LLNL/spack#25571]
### Steps to reproduce the issue

```console
spack install flux-sched
```

### Information on your system

On the OLCF Ascent cluster

```console
$ ./spack debug report
* **Spack:** 0.16.2-3997-37a1885
* **Python:** 3.7.0
* **Platform:** linux-rhel7-power9le
* **Concretizer:** original
```

### Additional information

[spack-build-env.txt](https://github.com/spack/spack/files/7033790/spack-build-env.txt)
[spack-build-out.txt](https://github.com/spack/spack/files/7033791/spack-build-out.txt)

When I add the following line to the flux-sched package.py file, it works.
```python
    depends_on("libedit")
```

@SteVwonde

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

**Comment by grondo** [comment]
Fixed by 22cfc19bcbc9d4df84dbd7f8023fbeb7de6ea94f?

**Comment by PhilipFackler** [comment]
yes, thank you.
